.github/workflows/pr.yml

```diff
@@ -0,0 +1,244 @@
+# This GitHub Actions workflow automates the process of
+# publishing dataset collections to a staging environment
+# It is triggered by a pull request to the main branch
+# that modifies any files within the ingestion-data/dataset-config/ directory
+# The workflow includes steps to
+#   - publish the datasets,
+#   - constantly updates the status of the workflow in the PR comment
+
+name: Publish collection to staging
+
+on:
+  pull_request:
+    branches: ['main']
+    paths:
+      # Run the workflow only if files inside this path are updated
+      # - ingestion-data/staging/dataset-config/*
+      - ingestion-data/testing/dataset-config/*
```

Comment:
Since we're going live with this in main. remove the testing path and use the `ingestion-data/staging/dataset-config/*` path instead?

Reply:
https://github.com/NASA-IMPACT/veda-data/issues/184 I updated this issue to capture that ask! @smohiudd